fd/diff.go

```diff
@@ -140,8 +162,8 @@ func Gradient(dst []float64, f func([]float64) float64, x []float64, settings *S
 	sendChan := make(chan fdrun, expect)
 	ansChan := make(chan fdrun, expect)
 
-	// Launch workers. Workers receive an index and a step, and compute the answer
-	for i := 0; i < settings.Workers; i++ {
```

Comment:
This explains why the tests previously took relatively long time to finish, one of them set `Workers = 1000`.